employee-service/src/test/java/com/rabbit/samples/employeeservice/unittests/services/EmployeeServiceUnitTest.java

```diff
@@ -4,74 +4,52 @@
 import com.rabbit.samples.employeeservice.persistence.repos.EmployeeRepository;
 import com.rabbit.samples.employeeservice.services.EmployeeService;
 import com.rabbit.samples.employeeservice.services.impl.EmployeeServiceImpl;
+import org.assertj.core.api.Assertions;
 import org.junit.Assert;
 import org.junit.Test;
 import org.junit.runner.RunWith;
 import org.mockito.BDDMockito;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
 import org.mockito.internal.verification.VerificationModeFactory;
+import org.mockito.junit.MockitoJUnitRunner;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.boot.test.context.TestConfiguration;
 import org.springframework.boot.test.mock.mockito.MockBean;
 import org.springframework.context.annotation.Import;
 import org.springframework.test.annotation.IfProfileValue;
 import org.springframework.test.context.junit4.SpringRunner;
 
+import static org.assertj.core.api.Assertions.*;
+import static org.mockito.BDDMockito.*;
 
-@IfProfileValue(name = "spring.testing.profile", values = {"all", "unit"})
```

Comment:
This line is required to distinguish if this test is unit, integration or whatever. I will test if this is working just with spring :)